fcgl/recommendation
# Database Implementation

## Problem
Need MongoDB in a container, and we need our database table modeled in python

## Solution
Create container for mongodb
Connect mongodb to project
Create Models for document representations

## Testing
1. run and build `docker-compose up --build`
2. Go to http://localhost:5000/ and confirm that all the api requests work
    * **Note**: None fo the `get` requests require you to pass in parameters
    * **Note**: The `post` requests require you to pass in parameters
    * **Note**: `merchant_product`, `merchant_category`, and `product` don't have post endpoints because I got lazy
    * **Note**: These API's are for testing purposes only, and as an example for how to work with the APIs in flask. In a production level implementation we would have to do data validation and error handling
